git-try-push/action.yml

```
@@ -26,6 +26,9 @@ inputs:
   force:
     description: Whether to force-with-lease push or not
     required: false
+  force-overwrite:
```

```suggestion
  force-master:
```

or something? I almost wonder if we want a `upstream_branch` that defaults to `origin/branch` but can be set to `master` for this case?

---

Yep. `origin_branch` maybe better?